src/ceph-daemon/ceph-daemon

```diff
@@ -2007,6 +2168,10 @@ def _get_parser():
     parser_deploy.add_argument(
         '--config-and-keyrings',
         help='JSON file with config and keyrings for the daemon and crash agent')
+    parser_deploy.add_argument(
+        '--mgr-list',
+        help='Comma separated list of ceph mgr IP address or names e.g. mgr-1,mgr2'
+    )
```

Comment:
i wonder if instead of adding this arg we change the --config-and-keyrings above to --config-json or something and bundle everything in there.  becuase for instance the grafana is going to take all kinds of random info from mgr/ssh.  And we could move some of the prometheus config that is hardcoded here into mgr/ssh too.

Reply:
Not sure about repurposing config-and-keyrings - the function returns a tuple of 3 and re very ceph specific. why not replace mgr-list and go for a generic --config-json this could be in the install payload and also the parameter for the reconfig payload?

Reply:
sounds good to me

Reply:
config-json added